sdk/src/org.graalvm.nativeimage/src/org/graalvm/nativeimage/hosted/RuntimeSerialization.java

```diff
@@ -54,6 +54,10 @@
 @Platforms(Platform.HOSTED_ONLY.class)
 public final class RuntimeSerialization {
 
+    public static void registerAllAssociatedClasses(Class<?> clazz) {
```

Comment:
Please add Javadoc with `@since` tag.

Reply:
Since which version? 23.0?

Reply:
I would guess so, depending on when this gets merged as well. Let's see what the GraalVM team says about it. Thanks for addressing this!